configs/components/fesom/Subjob_tripy.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#Home directory and reset of variable
+cd ~
+source ~/.bash_profile
```

Comment:
I would think it is cleaner to have the script not rely on anything set up in the user's `.bash_profile`. What is in your `.bash_profile` that you need to have here?